src/load_games.py

```diff
@@ -21,38 +64,47 @@ def load_games(path: str = 'resources/PGN_database',
             path to games database folder.
             Default: resources/PGN_database
 
-        is_rated : bool
-            filter rated games.
-            {True -> only rated,
-             False -> only casual,
-             None -> all games}
-            Default: None
-
     Returns
     -------
         List[dict]
             list of games in DB, sorted by date.
 
     """
 
+    fname = []
```

Comment:
Replace flatten of two maps for single list comprehension.

```suggestion
    fname = [os.path.join(folder, file) for folder, children, files in os.walk(base_folder) for file in files]
```